honcho/export/templates/upstart/master.conf

```diff
@@ -1,10 +1,5 @@
 pre-start script
```

Comment:
There's no need for the entire pre-start script block now, so perhaps you could remove that?